docking.js

```diff
@@ -44,6 +44,14 @@ const scrollAction = {
     SWITCH_WORKSPACE: 2
 };
 
+/* Array of all the docks created */
+let allDocks = [];
+
+/* status variable: true when the overview is shown through the dash
+ * applications button.
+ */
+let forcedOverview = false;
+
```

Comment:
If we have a dockManager object maybe these should be inside the dockManager (or some other class, not at the single dock level)